### PR TITLE
OEL-1283: Add update instructions for paragraphs move

### PR DESCRIPTION
**Pull request by donquixote**
[OEL-1283](https://citnet.tech.ec.europa.eu/CITnet/jira/browse/OEL-1283)

See also https://github.com/openeuropa/oe_bootstrap_theme/pull/210

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,3 +147,26 @@ To run the phpunit tests:
...
 docker-compose exec web ./vendor/bin/phpunit
 ```
+
+## Upgrade from older versions
+
+### Upgrade from 1.0.0-alpha5 and earlier
+
+Target release: You can upgrade directly to the version that contains this README.md.
+
+#### Paragraphs migration
```
Comment (drishu) [inline README.md]: makes more sense to me saying "Upgrading from version x to y" or "Upgrading to version y"

File: README.md [hunk 2]
```diff
@@ -147,3 +147,26 @@ To run the phpunit tests:
...
+
+Paragraphs-related theming and functionality has been moved from `oe_bootstrap_theme` to `oe_whitelabel`.
```
Comment (drishu) [inline README.md]: link oe_whitelabel

**Review by drishu — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -147,3 +147,26 @@ To run the phpunit tests:
 ```bash
 docker-compose exec web ./vendor/bin/phpunit
 ```
+
+## Upgrade from older versions
+
+### Upgrade from 1.0.0-alpha5 and earlier
+
+Target release: You can upgrade directly to the version that contains this README.md.
```
Comment (brummbar) [inline README.md]: Again, what does this mean?

File: README.md [hunk 4]
```diff
@@ -147,3 +147,26 @@ To run the phpunit tests:
...
+If you have the `oe_paragraphs` module enabled, you should:
+- Create a `hook_update_N()` or `hook_post_update_NAME()` in a custom module, to enable the `oe_whitelabel_paragraphs` module.
+- Run `drush updb` in your local installation.
+- Run `drush config:export`, to export the changes.
+- For deployment, make sure that `drush updb` runs _before_ `drush config:import`.
+
+If you did _not_ have `oe_paragraphs` enabled, but you want to do so now, you should:
+- Enable the new `oe_whitelabel_paragraphs` module in your local installation, ideally with `drush en`.
+- Export configuration, e.g. via `drush cex`. This should add the module to `core.extension.yml`.
+- For deployment, `drush config:import` will do the job. No update hook needed.
```
Comment (brummbar) [inline README.md]: Information NOT needed. We need to write HOW to do the upgrade from previous version. This is general information written in dozens of pages, no need to write it here again.

**Review by brummbar — CHANGES_REQUESTED** [review]
Not ok for me. We should add only the steps to be taken if we are updating from an old version AND we have oe_bt_paragraphs enabled. Then we need the enabling of the new module through update hook, and that's all.

**Review by drishu — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -147,3 +147,25 @@ To run the phpunit tests:
...
+
+### Upgrade to 1.0.0-alpha6
+
+#### Paragraphs migration
+
+Paragraphs-related theming and functionality has been moved from the [OpenEuropa Bootstrap base theme](https://github.com/openeuropa/oe_bootstrap_theme) to [OpenEuropa Whitelabel](https://github.com/openeuropa/oe_whitelabel).
+
+Special paragraphs fields that were introduced in `oe_bootstrap_theme_paragraphs` are being renamed in `oe_whitelabel_paragraphs`.
+
+If you have the `oe_paragraphs` module enabled, you should create a `hook_update_N()` or `hook_post_update_NAME()` in a custom module, to enable the `oe_whitelabel_paragraphs` module during deployment.
```
Comment (brummbar) [inline README.md]:
```suggestion
If you have the `oe_paragraphs` module enabled, you should create a `hook_post_update_NAME()` in your code, to enable the `oe_whitelabel_paragraphs` module during deployment.
```

**Review by brummbar — CHANGES_REQUESTED** [review]

**Review by brummbar — APPROVED** [review]

**Pull request merged**